test/test_template_targets.py

```
@@ -0,0 +1,33 @@
+    missing = []
+    for source in _python_sources():
+        text = source.read_text(encoding='utf-8')
+        for lineno, line in enumerate(text.splitlines(), 1):
+            for target in RENDER_RE.findall(line):
+                if not (TEMPLATE_ROOT / target).is_file():
+                    missing.append(
+                        f'{source.relative_to(PROJECT_ROOT)}:{lineno} -> {target}'
+                    )
+    assert not missing, (
+        'render_template() references templates that do not exist under '
+        f'web_interface/templates/:\n' + '\n'.join(missing)
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)